config/kubernetes/sig-docs/teams.yaml

```diff
@@ -42,9 +43,10 @@ teams:
     - smana
     - yastij
     privacy: closed
-  sig-docs-fr-owners:
-    description: Owners for French content
+  sig-docs-fr-reviews:
```

Comment:
@rlenferink sorry for the extra trouble :see_no_evil: ...but could you also add @oussemos to fr-owners and fr-reviews?

Ref: https://github.com/kubernetes/website/pull/12708 and https://github.com/kubernetes/org/issues/532

Reply:
Not a problem at all. I’ve updated this PR to include @oussemos as well ;)